Phase2/src/util/WordUtil.java

```diff
@@ -0,0 +1,43 @@
+package util;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class WordUtil {
+    private static ArrayList<String> stopWords = new ArrayList<>();
+
+    public static String extractRootWord(String word){
+        word = word.toLowerCase();
+        word = removeSymbols(word);
+        if (isStopWord(word))
+            return null;
+        return word;
+    }
+
+    private static String removeSymbols(String word) {
+        StringBuilder res = new StringBuilder();
+        for (int i = 0; i < word.length(); i++) {
+            if (Character.isAlphabetic(word.charAt(i)))
+                res.append(word.charAt(i));
+        }
+        return res.toString();
+    }
+
+    private static boolean isStopWord(String word) {
+        if (stopWords.size() == 0)
+            initStopWords();
+        return stopWords.contains(word);
+    }
+
+    private static void initStopWords() {
+        try (Scanner fileScanner = new Scanner(new File(Config.retrieveProperty("STOP_WORDS_PATH")))) {
```

Comment:
I don't have any idea on what is the best way to not hardcode this keys, but what I got from your opinion is that use another key-set to prevent hardcoding. I think enums cannot handle this problem, because we need something like a hash-map, to map some keys into some config keys. Besides that, using another key-set to map keys to config key is not interesting:

our approach: hardcode config keys -> get config properties
your approach: use another key-set -> get config keys -> get config properties

However instead of second approach, we can use this approach:
use a set of keys -> get config properties

But in this approach, we cannot use config file. So I think this would be the easiest way to use config file.